### PR TITLE
Java 37799 :- Upgrade to boot 3.2.2 for Spring Boot Libraries-2.

### PR DESCRIPTION
**Pull request by amit2103**

### CODE REVIEW

**Review by alvin577austria — APPROVED** [review]

File: spring-boot-modules/spring-boot-libraries-2/src/main/java/com/baeldung/annotation/scanner/AnnotationScannerApp.java [hunk 1]
```diff
@@ -1,12 +1,12 @@
 package com.baeldung.annotation.scanner;
 
-import org.jobrunr.autoconfigure.JobRunrAutoConfiguration;
+import org.jobrunr.spring.autoconfigure.JobRunrAutoConfiguration;
```
Comment (alvin577austria) [inline spring-boot-modules/spring-boot-libraries-2/src/main/java/com/baeldung/annotation/scanner/AnnotationScannerApp.java:3]: Does not seem to be used. Maybe we can remove this.

File: spring-boot-modules/spring-boot-libraries-2/src/main/java/com/baeldung/jobrunr/config/StorageProviderConfig.java [hunk 2]
```diff
@@ -1,18 +1,27 @@
 package com.baeldung.jobrunr.config;
 
 import org.jobrunr.jobs.mappers.JobMapper;
+import org.jobrunr.scheduling.JobScheduler;
 import org.jobrunr.storage.InMemoryStorageProvider;
 import org.jobrunr.storage.StorageProvider;
+import org.jobrunr.utils.mapper.jackson.JacksonJsonMapper;
+import org.springframework.beans.factory.annotation.Qualifier;
```
Comment (alvin577austria) [inline spring-boot-modules/spring-boot-libraries-2/src/main/java/com/baeldung/jobrunr/config/StorageProviderConfig.java:8]: Unused.

File: spring-boot-modules/spring-boot-libraries-2/src/main/java/com/baeldung/jobrunr/config/StorageProviderConfig.java [hunk 3]
```diff
@@ -1,18 +1,27 @@
...
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
+import org.springframework.web.client.RestTemplate;
```
Comment (alvin577austria) [inline spring-boot-modules/spring-boot-libraries-2/src/main/java/com/baeldung/jobrunr/config/StorageProviderConfig.java:11]: Unused.

File: spring-boot-modules/spring-boot-libraries-2/src/main/java/com/baeldung/kong/StockApp.java [hunk 4]
```diff
@@ -1,12 +1,12 @@
 package com.baeldung.kong;
 
-import org.jobrunr.autoconfigure.JobRunrAutoConfiguration;
+import org.jobrunr.spring.autoconfigure.JobRunrAutoConfiguration;
```
Comment (alvin577austria) [inline spring-boot-modules/spring-boot-libraries-2/src/main/java/com/baeldung/kong/StockApp.java:3]: Seems unused.

File: spring-boot-modules/spring-boot-libraries-2/src/main/java/com/baeldung/modulith/Application.java [hunk 5]
```diff
@@ -2,15 +2,15 @@
 
 import com.baeldung.modulith.product.ProductService;
 import com.baeldung.modulith.product.internal.Product;
-import org.jobrunr.autoconfigure.JobRunrAutoConfiguration;
+import org.jobrunr.spring.autoconfigure.JobRunrAutoConfiguration;
```
Comment (alvin577austria) [inline spring-boot-modules/spring-boot-libraries-2/src/main/java/com/baeldung/modulith/Application.java:5]: Unused.

**Review by alvin577austria — CHANGES_REQUESTED** [review]

File: spring-boot-modules/pom.xml [hunk 6]
```diff
@@ -53,8 +53,9 @@
         <module>spring-boot-jsp</module>
         <module>spring-boot-keycloak</module>
         <module>spring-boot-keycloak-2</module>
+
```
Comment (amit2103) [inline spring-boot-modules/pom.xml:56]: Intentional because of the libraries

**Review by alvin577austria — APPROVED** [review]

**Pull request merged**